app/src/main/res/navigation/mobile_navigation.xml

```diff
@@ -5,14 +5,14 @@
     android:id="@+id/mobile_navigation"
     app:startDestination="@+id/navigation_home">
 
+    <action android:id="@+id/action_logout"
+        app:destination="@+id/user_login" />
+
```

Comment:
Created a **global action** since the user should be able to log out from **anywhere**, not just a specific fragment